encrypted-media-respec.html

```diff
@@ -3611,7 +3612,7 @@
 
             <dt>User deletion of Key System storage</dt>
             <dd>
-              <p>User agents SHOULD present the user with a way to delete Key System storage for a specific <a def-id="origin"></a> or all origins.
+              <p>User agents SHOULD present the user with a way to specifically delete Key System storage, by <a def-id="origin"></a> or for all origins.
```

Comment:
I'm open to "or" being sufficient, but I don't think "all origins" encompasses "by origin." It might encompass "for an origin," but "by origin" means one at a time.

For example, "by origin" means the user can choose to delete Key System data just for example.com but not for foo.com and all other origins.